NNPTPZ1/ComplexNumber.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+
+namespace NNPTPZ1
+{
+
+    namespace Mathematics
+    {
+        public class ComplexNumber
+        {
+            public double RealPart { get; set; }
+            public float ImaginaryPart { get; set; }
+
+            public override bool Equals(object comparedObject)
+            {
+                if (comparedObject is ComplexNumber)
+                {
+                    ComplexNumber comparedComplexNumber = comparedObject as ComplexNumber;
+                    return comparedComplexNumber.RealPart == RealPart && comparedComplexNumber.ImaginaryPart == ImaginaryPart;
+                }
+                return base.Equals(comparedObject);
+            }
+
+            public readonly static ComplexNumber Zero = new ComplexNumber()
+            {
+                RealPart = 0,
+                ImaginaryPart = 0
+            };
+
+            public ComplexNumber Multiply(ComplexNumber multiplier)
+            {
+                ComplexNumber multiplicand = this;
+                return new ComplexNumber()
+                {
+                    RealPart = multiplicand.RealPart * multiplier.RealPart - multiplicand.ImaginaryPart * multiplier.ImaginaryPart,
+                    ImaginaryPart = (float)(multiplicand.RealPart * multiplier.ImaginaryPart + multiplicand.ImaginaryPart * multiplier.RealPart)
+                };
+            }
+
+            public ComplexNumber Add(ComplexNumber addend)
+            {
+                ComplexNumber augend = this;
+                return new ComplexNumber()
+                {
+                    RealPart = augend.RealPart + addend.RealPart,
+                    ImaginaryPart = augend.ImaginaryPart + addend.ImaginaryPart
+                };
+            }
+
+            public ComplexNumber Divide(ComplexNumber Divisor)
+            {
+                var numerator = Multiply(new ComplexNumber() { RealPart = Divisor.RealPart, ImaginaryPart = -Divisor.ImaginaryPart });
+                var denominator = Divisor.RealPart * Divisor.RealPart + Divisor.ImaginaryPart * Divisor.ImaginaryPart;
+
+                return new ComplexNumber()
+                {
+                    RealPart = numerator.RealPart / denominator,
+                    ImaginaryPart = (float)(numerator.ImaginaryPart / denominator)
+                };
+            }
+
+            public double GetAngleInDegrees()
```

Comment:
name is not corresponding to implementation

Reply:
I changed the method body instead of name - so it should actually get angle in degrees (if the * 180 / Math.PI conversion to degrees is correct). Nontheless in next commit I also added method GetAngleInRadians.